cmd/snap/cmd_auto_import.go

```diff
@@ -100,8 +134,19 @@ func autoImportFromAllMounts() (int, error) {
 
 	added := 0
 	for _, cand := range cands {
-		if err := ackFile(cand); err != nil {
+		err := ackFile(cand)
+		// HORRIBLE: string compare OMG
+		// the server is not ready yet
+		if err != nil && strings.Contains(err.Error(), "cannot communicate with server") {
```

Comment:
It seems nice that we're checking for this specific case instead of spooling on every error, but can we please make this a simple error type, such as:
```
type ConnectionError struct {
        err error
}

func (e *ConnectionError) Error() string {
        return fmt.Errorf("cannot communicate with server: %v", e.err)
}
...
return &ConnectionError{err}
```